code/modules/mob/living/carbon/xenomorph/charge_crush.dm

```diff
@@ -392,6 +392,9 @@
 		unbuckle_mob(m)
 	return (CHARGE_SPEED(charge_datum) * 20) //Damage to inflict.
 
+/obj/vehicle/unmanned/pre_crush_act(mob/living/carbon/xenomorph/charger, datum/action/xeno_action/ready_charge/charge_datum)
+	return (CHARGE_SPEED(charge_datum) * 10)
```

Comment:
> wdym reduced , they take double damage compared to other items

And half damage compared to their living counterparts `var/crush_living_damage = 20`

Reply:
they take almost 50 damages, what are you on?

Reply:
The point is moot anyway cause I have literally never hit or seen any class of unmanned vehicle hit by a crusher outside of a 1 wide hallway where funky things happen because they can't ever be "run over" like humans. You can resolve this.

Reply:
How is hitting them related in the slightest with being run  over? You can't run over something you can't hit in the first plae

Reply:
Yeah exactly. The only time you will see hits happen is when they should also be run over except that is not possible so either the charge gets stopped or they just keep getting crushed until dead. Not sure I can be clearer about it but if you actually care about the issue you can ping me about it. I get the feeling it's not a concern tho.

Reply:
Just as I suspected the only place you can really hit, only because they can't side step, it's almost always guaranteed to kill.
Look how many hits they survive too I counted 5 to 7 charge hit sound effects! Wow they are strong stuff! Recorded just now from the latest PR commit.
![5dUJIzGXHS](https://user-images.githubusercontent.com/64715958/127706158-0eebc3cd-1832-4f94-8f02-624a4261ecbc.gif)
```
Young Crusher (Oops) roars!
XN-43-H combat droid 4 blows apart!
We crush XN-43-H combat droid 4!
```